pytorch_lightning/loops/epoch/evaluation_epoch_loop.py

```diff
@@ -269,23 +273,6 @@ def _on_evaluation_batch_end(self, output: Optional[STEP_OUTPUT], **kwargs: Any)
 
         self.trainer.logger_connector.on_batch_end()
 
-    def _build_kwargs(self, batch: Any, batch_idx: int, dataloader_idx: Optional[int]) -> Dict[str, Union[Any, int]]:
```

Comment:
why don't we keep this? this was useful to override

Reply:
I am fine either way, but I liked this structure.

Reply:
Done